### ISSUE TITLE
Prompt for 'wk' and 'writing_rec'

### ISSUE THREAD

**Issue by XinXU-USTC** [hkust-nlp/felm#1]
In your code, it seems that you use 'sci' prompts for 'wk' and 'writing_rec' domain. (line 131- line 139 in eval.py) 
<img width="1069" alt="image" src="https://github.com/hkust-nlp/felm/assets/67425980/e9edd3a9-7fd0-460b-855f-cd98bdedef0f">
I think maybe there are typos shown in this screen shot. 
I think the prompt may influence the final outcomes. May I ask that the results reported in the paper use which prompts for the 'wk' and 'writing_rec' domains?

**Comment by XinXU-USTC** [comment]
If I understand your codes correctly, something strange will happen if the method is 'link' or 'content': you are still counting the numbers of examples in domain 'math' and 'reasoning'. Don't you think we should filter these two domains and only count the numbers in the other domain?

**Comment by shiqichen17** [comment]
Thank you for your attentive observations and sorry for our oversights!
First, we would like to clarify that the code to obtain the results in the paper is a bit different from the current version of the code in this repo — we did some cleaning and reorganization of the code for open-sourcing which turns out to have introduced some typos.
For the prompt question, you are right, this is a typo bug introduced during the rebuttal phase to run the segment-based results for Vicuna. This bug has impacted only the segment-based results of Vicuna on the “wk” and “writing_rec” domains (The GPTs evaluation code/results and claim-based eval of Vicuna code/results are correct in our local version). We have rectified this issue by correcting the code and re-runing the results. And we are updating the results in our Arxiv version now, ensuring accurate Vicuna-30B results. (The overall conclusion remains unchanged.)
As for the Link/Doc augmented methods, we account for the vanilla numbers for math and reasoning domains to compute the overall results of Link/Doc, as explained by the caption of Table 4 in the paper. This is to ensure the overall results of Link/Doc are comparable to the overall results of other evaluation schemes such as vanilla and cot. There is no bug in our reported results on this part. (We performed additional calculations to accurately sum up all the results.) We have now updated the code to ensure clarity and accuracy.
Apologies for the confusion and mistakes again!
